### PR TITLE
Adding mint module for minting NFTs

### PR DESCRIPTION
**Pull request by bengriffin1**
### 📦 Pull Request

Adding a feature for NFT Minting + Delivery. Adds a new "Mint" module with 2 functions, for both ERC721 and ERC1155 minting.

### ✅ Fixed Issues

None

### 🚨 Test instructions

yarn run test

### ⚠️ Don't forget to add a [semver](https://semver.org/) label!
`minor`

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,14 @@
+# v1.9.0 (Mon Mar 20 2023)
+
+#### 🚀 Enhancement
+
+- New module for NFT minting + delivery [#?](https://github.com/magiclabs/magic-admin-js/pull/?) ([@bengriffin1](https://github.com/bengriffin1))
```
Comment (justinnout) [inline CHANGELOG.md:5]:
```suggestion
- New module for NFT minting + delivery [#?](https://github.com/magiclabs/magic-admin-js/pull/105) ([@bengriffin1](https://github.com/bengriffin1))
```

File: src/core/sdk-exceptions.ts [hunk 2]
```diff
@@ -63,3 +63,10 @@ export function createExpectedBearerStringError() {
     'Expected argument to be a string in the `Bearer {token}` format.',
   );
 }
+
+export function mintingError() {
+  return new MagicAdminSDKError(
+    ErrorCode.MintingError,
+    'There was an error while minting. Check your contract ID and token ID (ERC1155 only)'
```
Comment (justinnout) [inline src/core/sdk-exceptions.ts:70]: Seems this also supports `ERC721`?
```suggestion
    'There was an error while minting. Check your contract ID and token ID (ERC721 and ERC1155 only)'
```

File: test/spec/modules/mint/mint721.spec.ts [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+    error: 'bar_456'
+});
+
+test('Successfully POSTs to 721 minting endpoint', async () => {
+  const sdk = createMagicAdminSDK('https://example.com');
+
+  const postStub = jest.fn().mockImplementation(() => successReq);
+  (post as any) = postStub;
+
+  await expect(sdk.mint.startMint721('0xfoo', 1, '0xbar')).resolves.not.toThrow();
```
Comment (justinnout) [inline test/spec/modules/mint/mint721.spec.ts:20]: Can we assert on the result from this call?

File: test/spec/modules/mint/mint1155.spec.ts [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+    error: 'bar_456'
+});
+
+test('Successfully POSTs to 1155 minting endpoint', async () => {
+  const sdk = createMagicAdminSDK('https://example.com');
+
+  const postStub = jest.fn().mockImplementation(() => successReq);
+  (post as any) = postStub;
+
+  await expect(sdk.mint.startMint1155('0xfoo', 1, '0xbar', 0)).resolves.not.toThrow();
```
Comment (justinnout) [inline test/spec/modules/mint/mint1155.spec.ts:20]: Can we assert the result from this call?

**Review by justinnout — COMMENTED** [review]
Looks good, just a few nits

File: src/core/sdk-exceptions.ts (hunk 2 above)
Comment (bengriffin1) [reply src/core/sdk-exceptions.ts:70]: Oh this is bad phrasing -- the tokenID is only present / required on ERC1155. I'll update to be more clear

File: test/spec/modules/mint/mint1155.spec.ts (hunk 4 above)
Comment (bengriffin1) [reply test/spec/modules/mint/mint1155.spec.ts]: Good call, updated in latest

**Review by justinnout — APPROVED** [review]
LGTM!

**Pull request merged**